### DISCUSSION TITLE
We can't log in to the admin portal

### DISCUSSION THREAD

**Discussion by pera4698** [globaleaks/globaleaks-whistleblowing-software#4073] [Q&A]
### What version of GlobaLeaks are you using?

I can't tell you the version as I can't log in. We created it on 04/12/2023 and it is no longer updated

### What browser(s) are you seeing the problem on?

All

### What operating system(s) are you seeing the problem on?

Windows

### Describe the issue

Good morning, we are no longer able to access the portal in the admin interface of all our globalleaks. We have configured double authentication, the password works but the verification code doesn't work. We are stuck.

**Comment by evilaliv3** [comment]
@pera4698 if you could update to version 4.15.4 you will be able to disable two factor authentication using the ```gl-admin``` command line tool
